### ISSUE TITLE
Jolokia support

### ISSUE THREAD

**Issue by akadoya** [strapdata/elassandra#121]
**Elassandra version**: elassandra-2.4.5.3

**JVM version**: 1.8.0_131

**OS version**: Ubuntu 16.04

**Description of the problem including expected versus actual behavior**:

Cannot use Jolokia with elassandra due to jar hell error when starting up elassandra.

**Steps to reproduce**:
 1. Put Jolokia.jar in ${CASSANDRA_HOME}/lib
 2. Start Elassandra
 3. It'll fail with the errors shown below.

**Provide logs (if relevant)**:

```
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:421)
	at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:115)
	at org.elasticsearch.node.Node.<init>(Node.java:161)
	at org.elasticsearch.node.Node.<init>(Node.java:143)
	at org.elasticsearch.node.NodeBuilder.build(NodeBuilder.java:153)
	at org.apache.cassandra.service.ElassandraDaemon.setup(ElassandraDaemon.java:339)
	at org.apache.cassandra.service.ElassandraDaemon.activate(ElassandraDaemon.java:143)
	at org.apache.cassandra.service.ElassandraDaemon.main(ElassandraDaemon.java:432)
Caused by: java.lang.IllegalStateException: jar hell!
class: org.json.simple.ItemList
jar1: /opt/elassandra/elassandra-2.4.5.3/lib/jolokia-jvm-1.3.7-agent.jar
jar2: /opt/elassandra/elassandra-2.4.5.3/lib/json-simple-1.1.jar
```

**Comment by vroyer** [comment]
Hi,
For code quality insurance, elasticsearch use jarHell to ensure that the classpath does not contains several copies of the same class, potentially in different version that could cause unexpected results depending on the class loading order.  

Here, jolokia-jvm-1.3.7-agent.jar contains a class org.json.simple.ItemList already available in json-simple-1.1.jar. Jokolia should remove this class from its jar and have a dependency to json-simple (or at least rename its version of org.json.simple.ItemList). 
Thanks.

**Comment by akadoya** [comment]
understood. thank you for the explanation. I have submitted an issue ticket to Jolokia side instead.
